packages/tools/kadena-cli/src/tx/utils/txDisplayHelper.ts

```diff
@@ -34,53 +36,113 @@ export async function printTx(
 }
 
 export function txDisplayTransaction(
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  obj: any,
-  signedTransactionFiles: string[],
+  data: { transactions: ISubmitResponse[] },
+  files: string[],
   header: string = '',
-  baseIndent: number = 2, // Base indentation level for all lines
+  baseIndent: number = 2,
 ): void {
   if (header !== '') {
     displaySeparator();
     log.info(`  ${log.color.black(header.padEnd(formatLength - 2))}`);
     displaySeparator();
   }
 
-  if (obj === null || obj === undefined) {
+  if (
+    data === undefined ||
+    data.transactions === undefined ||
+    data.transactions.length === 0
+  ) {
     log.info('Transaction '.repeat(baseIndent) + log.color.green('null'));
     return;
   }
 
+  data.transactions.forEach((item: ISubmitResponse, i: number) => {
+    displayCustomTransactionInfo(
+      {
+        fileName: files[i],
+        transactionHash: item.transaction.hash,
+      },
+      baseIndent,
+    );
+    log.info('\n');
+    if (item.response) {
+      log.info(log.color.black(`${' '.repeat(baseIndent)}Response:`));
```

Comment:
```suggestion
      log.info(log.color.black(`${' '.repeat(baseIndent)}Response:`));
```
can't we use padStart instead of using `black` color with indentation